bin/prettier.js

```diff
@@ -145,7 +146,10 @@ function getTrailingComma() {
   }
 }
 
+const r = require;
 const options = {
+  astTransform: argv["ast-transform"] &&
```

Comment:
Yeah... it is pretty deep in the internals, but I'd argue that it's still useful enough to provide a way for people to do it. Nevertheless, I'll rename this to `transform`, and maybe propose `docTransform` separately later.